docker-compose.dev.yml

```diff
@@ -111,4 +111,4 @@ services:
 
   cvat_redis_ondisk:
     ports:
-      - '6479:6379'
+      - '6479:6666'
```

Comment:
The "6479" number can probably also be replaced with 6666 everywhere; there's not much reason to keep it if the internal port number is different.